### ISSUE TITLE
EnderChest is sometimes not visible

### ISSUE THREAD

**Issue by sero583** [Nukkit/Nukkit#1822]
### Issue Description

Place a EnderChest by plugin or normal, for me it's sometimes not visible not even after re-log.

### OS and Versions

* Nukkit Version: This server is running Nukkit 1.0dev ???(Apple)?(Pie)? implementing API version 1.0.0 for Minecraft: PE v1.1.3 (protocol version 113) 

* Java Version: openjdk version "1.8.0_131"
OpenJDK Runtime Environment (build 1.8.0_131-8u131-b11-0ubuntu1.16.04.2-b11)
OpenJDK 64-Bit Server VM (build 25.131-b11, mixed mode)
```
??
```

* Host Configuration: 

| Item | Value |
|:----:|:-----:|
| Host OS | ?? |  
| Memory(RAM) | ?? GB | 
| Storage Size | ?? GB | 
| Storage Type | ?? | 
| CPU Type | ?? | 
| CPU Core Count | ?? cores ?? threads | 
| Upstream Bandwidth | ?? | 

* Client Configuration: 

| Item | Value |
|:----:|:-----:|
| Client Edition | Android | 
| Client Version | 1.1.0 | 

![screenshot_20170728-003434](https://user-images.githubusercontent.com/12701010/28695111-80266984-732d-11e7-9772-1759dad072e7.png)
![screenshot_20170728-003439](https://user-images.githubusercontent.com/12701010/28695123-953043a4-732d-11e7-92b2-e5af63bb3cb1.png)

**Comment by PetteriM1** [comment]
Can you open it?

**Comment by sero583** [comment]
No, it acts for me like invisible bedrock but it ain't because you see the bedrock particles but I can break it. I guess maybe its an client side issue
